### DISCUSSION TITLE
31F279A8-BE40-FB0E-1D42-BBE5CADD6EDF

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#90] [General]
# Earth: Rings

This addon adds fictional rings around the Earth.

https://celestia.mobi/addon?item=31F279A8-BE40-FB0E-1D42-BBE5CADD6EDF&lang=lt

**Comment by Einaras553** [comment]
This app is so underrated ngl!

**Comment by hiimdrram** [comment]
Sosytem

**Comment by TyiVR** [comment]
The rings are hella realistic
